registration/include/pcl/registration/gicp6d.h

```diff
@@ -45,38 +45,7 @@
 #include <pcl/point_cloud.h>
 #include <pcl/point_representation.h>
 #include <pcl/point_types.h>
-
-namespace pcl {
-struct EIGEN_ALIGN16 _PointXYZLAB {
-  PCL_ADD_POINT4D; // this adds the members x,y,z
-  union {
-    struct {
-      float L;
-      float a;
-      float b;
-    };
-    float data_lab[4];
-  };
-  PCL_MAKE_ALIGNED_OPERATOR_NEW
-};
-
-/** \brief A custom point type for position and CIELAB color value */
-struct PointXYZLAB : public _PointXYZLAB {
-  inline PointXYZLAB()
-  {
-    x = y = z = 0.0f;
-    data[3] = 1.0f; // important for homogeneous coordinates
-    L = a = b = 0.0f;
-    data_lab[3] = 0.0f;
-  }
-};
-} // namespace pcl
-
-// register the custom point type in PCL
-POINT_CLOUD_REGISTER_POINT_STRUCT(
-    pcl::_PointXYZLAB,
-    (float, x, x)(float, y, y)(float, z, z)(float, L, L)(float, a, a)(float, b, b))
-POINT_CLOUD_REGISTER_POINT_WRAPPER(pcl::PointXYZLAB, pcl::_PointXYZLAB)
+#include <pcl/point_types_conversion.h>
```

Comment:
Is it necessary to include this here or can this be moved to gicp6d.cpp?

Reply:
Yes, it would be more explicit if we include header in the file where we call the functions. I'll move it to gicp6d.cpp. :blush: Thank you!